### ISSUE TITLE
Unable to reach url /socket.io/1/?... using express on nodester

### ISSUE THREAD

**Issue by juzerali** [Flotype/now#190]
I have tried creating a demo chat app on nodester. Although this app is working on my Windows 7 local, its not working on nodester. I have also asked a [question on stackoverflow](http://stackoverflow.com/questions/9934202/url-socket-io-1-404-not-found-nowjsnodesterexpress-chat-app) about this.

This is the error I am getting in browser's javascript console:

``` javascript
GET http://dlq.nodester.com/socket.io/1/?t=1333056376218 404 (Not Found) socket.io.js:1628
Socket.handshake socket.io.js:1628
Socket.connect socket.io.js:1667
Socket socket.io.js:1526
io.connect socket.io.js:91
scriptLoaded


This is my package.json
```

``` javascript
{
    "name": "application-name"
  ,  “node”:”v0.6.12”,
  , "version": "0.0.1"
  , "private": true
  , "dependencies": {
      "express": "2.5.8"
    , "jade": ">= 0.0.1"
    , "now": "*"
    , "socket.io: "*"
  }
}


Any ideas what might be the issue?
```

**Comment by steveWang** [comment]
Express currently breaks NowJS file-serving. Not sure how to fix.

**Comment by juzerali** [comment]
I don't think it is express that is breaking, since it is running fine on my local and on different devices on my private LAN. And on both nodester and my local I have Express 2.5.8 installed. It might have something to do with Operating System. I work on Windows 7, not sure about nodester's OS.

**Comment by FaisalAbid** [comment]
Hey, @juzerali - Did you get ever get this issue solved? I'm having the exact same problem.

**Comment by juzerali** [comment]
@FaisalAbid  It has been a long time. I remember I did got around this issue. Not remember how though.

**Comment by topheman** [comment]
@FaisalAbid @juzerali any luck remembering your work around ? I have the same problem ....
I get the socket.io.js client side script but the next request made is to : http://http/socket.io/1/?t=1349987996367  ...

**Comment by FaisalAbid** [comment]
Unfortunately, I never got it to work, and didn't spend too much time on it since it was dropped from scope.

**Comment by FaisalAbid** [comment]
However from what I can see, AppFog bought Nodestr, and this wasn't working on Nodester, so it could be something on their end. I know AppFog does not support websockets. Perhaps that has to do with something, even though Socket.io should fall back to polling if im not mistaken.

**Comment by topheman** [comment]
I found out, debugging the io.socket.js . It was my mistake ... I had different host configuration (local and nodester).

I had io.connect('http://'+hostname+':'port');

And i configured hostname = 'http://mydomainonnodester'

So, it tried to do io.connect('http://http://mydomainonnodester') ...

It works now ...

**Comment by juzerali** [comment]
This project is dead or what? No commits since last 8 months?

**Comment by riturajratan** [comment]
facing same issue can someone tell me that how to resolve it properly ?
